src/data/structure/Recommendation.java

```diff
@@ -0,0 +1,33 @@
+package data.structure;
+
+/**
+ * Represents a recommendation based on a UserProfile with its associated Preferences
+ *    (i.e. the user's preferences).  Contains the Preference that is most correlated
+ *      with the user's preferences as well as total correlation score.
+ * @author benjaminposerow
+ *
+ */
+public class Recommendation {
+	private Preference correlatedPreference;
+	private UserProfile user;
```

Comment:
If you feel strongly I will remove.  But a Recommendation doesn't make sense outside of the UserProfile for whom it is targeted.   So it seems useful to include the UserProfile to differentiate it from a Recommendation that might potentially have the exact same score and Preference but a different UserProfile (yes, this could occur!)